### PR TITLE
Increase timeout to download plugins

### PR DESCRIPTION
**Pull request by gjambet**

### CODE REVIEW

File: sonar-plugin-api/src/main/java/org/sonar/api/utils/HttpDownloader.java [hunk 1]
```diff
@@ -33,8 +33,20 @@
...
+   *   {@link #DEFAULT_READ_TIMEOUT_IN_MILLISECONDS}
+   *   {@link #DEFAULT_CONNECT_TIMEOUT_IN_MILLISECONDS}
+   *
+   * @deprecated since 7.0
+   */
+  @Deprecated
   public static final int TIMEOUT_MILLISECONDS = 20 * 1000;
 
+  public static final int DEFAULT_READ_TIMEOUT_IN_MILLISECONDS = 60 * 1000;
+  public static final int DEFAULT_CONNECT_TIMEOUT_IN_MILLISECONDS = TIMEOUT_MILLISECONDS;
```
Comment (SonarTech) [inline sonar-plugin-api/src/main/java/org/sonar/api/utils/HttpDownloader.java]: MINOR Remove this use of "TIMEOUT_MILLISECONDS"; it is deprecated. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

File: sonar-plugin-api/src/main/java/org/sonar/api/utils/HttpDownloader.java [hunk 2]
```diff
@@ -33,8 +33,20 @@
...
+
+  /**
+   * Catch-all default timeout, replaced by
+   *   {@link #DEFAULT_READ_TIMEOUT_IN_MILLISECONDS}
+   *   {@link #DEFAULT_CONNECT_TIMEOUT_IN_MILLISECONDS}
+   *
+   * @deprecated since 7.0
+   */
+  @Deprecated
   public static final int TIMEOUT_MILLISECONDS = 20 * 1000;
```
Comment (SonarTech) [inline sonar-plugin-api/src/main/java/org/sonar/api/utils/HttpDownloader.java]: INFO Do not forget to remove this deprecated code someday. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 1 minor
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* ![INFO][INFO] 1 info

Watch the comments in this conversation to review them.

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'

File: sonar-plugin-api/src/main/java/org/sonar/api/utils/HttpDownloader.java [hunk 3]
```diff
@@ -33,8 +33,20 @@
...
 
+  public static final int DEFAULT_READ_TIMEOUT_IN_MILLISECONDS = 60 * 1000;
```
Comment (henryju) [inline sonar-plugin-api/src/main/java/org/sonar/api/utils/HttpDownloader.java]: You could add `@since 7.0`

Comment (gjambet) [reply sonar-plugin-api/src/main/java/org/sonar/api/utils/HttpDownloader.java]: +1

**Pull request merged**